client/kubernetes/src/main/java/org/jolokia/kubernetes/client/KubernetesJmxConnector.java

```diff
@@ -71,12 +72,15 @@ protected J4pClient expandAndProbeUrl(KubernetesClient client,
       if (POD_PATTERN.matcher(proxyPath).matches()) {
         final Matcher matcher = POD_PATTERN.matcher(proxyPath);
         if (matcher.find()) {
-          String namespace = matcher.group(1);
-          String podPattern = matcher.group(2);
-          String path = matcher.group(3);
+          String namespace = matcher.group("namespace");
```

Comment:
Note, this requires Java 1.7 , I can happily replace with numeric groups, however was not sure if Java 1.6 is feasible anymore (?)

Reply:
We abandoned Java 1.6 already afair (my gosh, really not deep in Jolokia anymore :(.

Lets keep it ...